src/components/Worktray/Worktray.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+
+/**
+ * The WorkTray component, that can be used to display a user's current work items.
+ *
+ * @noInheritDoc
+ */
+
+export interface WorkTrayProps {
+  /**
+   * Array of {@link WorkTrayRow} that defines the number of rows within the Work Tray
+   */
+  rows: WorkTrayRow[];
+  /**
+   * Array of {@link WorkTrayColumn} that defines the number of columns within the Work Tray
+   */
+  columns: WorkTrayColumn[];
+  /**
+   * Callback function used to reassign a work item to another user. If this function returns true, the item will be removed from the work tray.
+   */
+  reassignWorkItem: (workItemId: string) => boolean;
+  /**
+   * Callback function used to cancel a work item. If this function returns true, the item will be removed from the work tray.
```

Comment:
Not sure if cancelling a work item should ultimately result in it disappearing or just entering a cancelled state. Probably doesn't matter yet and we can bottom it out in the cancellation tickets, but worth thinking about.